lib/cli/handle-error.js

```diff
@@ -118,43 +127,68 @@ module.exports = async (exception, options = {}) => {
       ' For debugging logs, run again after setting the',
       ' "SLS_DEBUG=*" environment variable.',
     ].join('');
-    consoleLog(chalk.red(debugInfo));
-    consoleLog(' ');
+    legacy.consoleLog(chalk.red(debugInfo));
+    legacy.consoleLog(' ');
   }
 
   const platform = process.platform;
   const nodeVersion = process.version.replace(/^[v|V]/, '');
-
-  consoleLog(chalk.yellow('  Get Support --------------------------------------------'));
```

Comment:
Why these logs are removed? 

I believe we should simply change the interface with which logs which have modern counterparts are written, and do not remove them

Reply:
This is not removed, but rather reorganized a little bit in the file as I needed to resolve some of the values earlier in order to properly print new outputs. It's just Github shows it in a weird way. It starts on line 172 now

Reply:
I see, yes, that makes sense